Jenkinsfile.release

```diff
@@ -9,12 +9,14 @@ void tryAddKnownHost(String hostUrl){
 
 pipeline {
     agent {
-        docker { image 'pegasyseng/pantheon-build:0.0.5-jdk8' }
+        docker { image 'pegasyseng/pantheon-build:0.0.5-jdk11' }
     }
     parameters {
         string(name: 'BRANCH_NAME', defaultValue: 'master', description: '[MANDATORY] The name of the branch to create the release from')
-        string(name: 'RELEASE_VERSION', defaultValue: '', description: '[OPTIONAL] When empty: defaults to the current project version')
+        string(name: 'RELEASE_VERSION', defaultValue: '1.x', description: '[MANDATORY] The version you are deploying')
         string(name: 'NEXT_VERSION', defaultValue: '', description: '[OPTIONAL] When empty: defaults to next patch version after current project version')
+        string(name: 'DOCKER_ADDITIONAL_TAGS', defaultValue: '', description: '[OPTIONAL] When empty: you get the release version as a tag only')
+        booleanParam(name: 'PUSH_RELEASE', defaultValue: true, description: '[OPTIONAL] Push the docker images to dockerhub')
```

Comment:
Is there a reason we'd release without releasing the docker images?

Reply:
The push was originally so we could trigger a test build without pushing it to servers. Might no longer make sense

Reply:
I've removed this param now